packages/database/src/migrations/20210211051517-RenameArithmeticIndicators-modifies-data.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+
+var dbm;
+var type;
+var seed;
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function (options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+exports.up = async function (db) {
+  await db.runSql(`
+    UPDATE indicator
+    SET builder = 'arithmeticAnalytic'
+    WHERE builder = 'arithmetic';
+  `);
+};
```

Comment:
Now that indicator can fetch events or analytics as inputs, we want to make this more obvious in the indicator names. So we want to change `arithmetic` to something more specific. I'm not sure if this is a good name. I'm thinking between 3 options:
- arithmeticAnalytic
- analyticArithmetic
- runAnalyticArithmetic

I went with `arithmeticAnalytic` and `countEvent` to be more consitent with how we name data builders... But any suggestions would be appreciated!